userspace/falco/app/actions/load_rules_files.cpp

```diff
@@ -66,12 +66,14 @@ falco::app::run_result falco::app::actions::load_rules_files(falco::app::state&
 	}
 
 	std::string err = "";
+	falco_logger::log(falco_logger::level::INFO, "Loading rules from:\n");
```

Comment:
Follow same behavior as Falco config files loading, ie: print a first line with `Loading rules from` and then proceed to list all loaded rules with their schema validation status each on a new line.